test/js-native-api/test_string/test.js

```diff
@@ -130,6 +130,19 @@ assert.strictEqual(test_string.TestPropertyKeyUtf16AutoLength(str6), str6);
 assert.strictEqual(test_string.Utf16Length(str6), 5);
 assert.strictEqual(test_string.Utf8Length(str6), 14);
 
+const str7 = 'propertyKey';
```

Comment:
Please make the JS tests for the new property creation to follow the same pattern as in the PR #50282.